01-atoms/buttons/buttons.stories.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import Button from './Button.component';
+
+/**
+ * Storybook Definition.
+ */
+export default { title: 'Atoms/Button' };
+
+export const All = () => (
+  <>
+    <Button>Button Default</Button>
+    <br />
```

Comment:
I don't have a design eye, but this 'All' story is very hard to look at with the buttons butted up against each other. Maybe not right now, but at some point, we could look for places to enhance our demo components.